test/verify/check-machines-lifecycle

```diff
@@ -98,30 +102,39 @@ class TestMachinesLifecycle(VirtualMachinesCase):
         wait(lambda: "login as 'cirros' user." in self.machine.execute("cat {0}".format(args["logfile"])), delay=3)
 
         # Send Non-Maskable Interrupt (no change in VM state is expected)
-        self.performAction("subVmTest1", "sendNMI")
+        b.click("#vm-subVmTest1-action-kebab button")
+        b.click("#vm-subVmTest1-sendNMI")
 
         b.wait(lambda: "NMI received" in self.machine.execute("cat {0}".format(args["logfile"])))
 
         # pause
-        self.performAction("subVmTest1", "pause")
+        b.click("#vm-subVmTest1-action-kebab button")
+        b.click("#vm-subVmTest1-pause")
+        b.wait_in_text("#vm-subVmTest1-state", "Paused")
 
         # resume
-        self.performAction("subVmTest1", "resume")
+        b.click("#vm-subVmTest1-action-kebab button")
+        b.click("#vm-subVmTest1-resume")
+        b.wait_in_text("#vm-subVmTest1-state", "Running")
 
         # reboot
         self.machine.execute("echo '' > {0}".format(args["logfile"]))
-        self.performAction("subVmTest1", "reboot")
+        b.click("#vm-subVmTest1-action-kebab button")
+        b.click("#vm-subVmTest1-reboot")
         wait(lambda: "reboot: Power down" in self.machine.execute("cat {0}".format(args["logfile"])), delay=3)
         b.wait_in_text("#vm-subVmTest1-state", "Running")
 
         # force reboot
         self.machine.execute("echo '' > {0}".format(args["logfile"]))
-        self.performAction("subVmTest1", "forceReboot")
+        b.click("#vm-subVmTest1-action-kebab button")
```

Comment:
This is clearly a rebase error :) And all the above test code, reverted a reverted commit.